modules/update-service-graph-data.adoc

```diff
@@ -0,0 +1,31 @@
+[id="update-service-graph-data_{context}"]
+= Creating the OpenShift Update Service graph data container image
```

Comment:
The section is not closely connected with section **Installing the OpenShift Update Service Operator**. It would be better to make it an individual section rather than a sub-section of **Installing the OpenShift Update Service Operator**.

Reply:
moved it